setup.py

```diff
@@ -31,6 +40,6 @@
     include_package_data=True,
     install_requires=requires,
     extras_require=requires_optional,
-    python_requires=">=3.8.0",
+    python_requires=">=3.6.1",
```

Comment:
Same situation as future-annotations. Remove this for now and we'll keep working on it separately.